docs/en/stack/ml/anomaly-detection/anomaly-detection-scale.asciidoc

```diff
@@ -0,0 +1,226 @@
+
+If you have a large amount of data that requires a short bucket span, consider 
+splitting it up to avoid having high cardinality. The combination of short 
+bucket span and high cardinality is a resource-intensive situation that you may 
+want to avoid.
+
+
+[[forecasting]]
+== Understanding the cost of forecasting
+
+There are two main factors to consider when you create a forecast: indexing load 
+and memory usage.
+
+Forecasting writes a new document to the result index for every forecasted 
+element of the job. Practically, it means that if you have a bucket span of 5 
+minutes and you are forecasting one hour to the future, you have twelve 
+documents for every job partition that is being predicted. For jobs with high 
+partition field cardinality, forecasting results in a high number of documents 
+being written out to the result index, which will add a high load to your data 
+nodes.
+
+The memory usage of a forecast is restricted to 20 MB by default. From 7.9, you 
+can extend this limit by setting `max_model_memory` to a higher value where the 
+maximum is 40% of the memory limit of the {anomaly-job} or 500 MB. If the 
+forecast needs more memory than the provided value, it spools to disk. Forecasts 
+that would take more than 500 MB to run won't start because this is the maximum 
+limit of disk space that a forecast is allowed to use. Jobs with high memory 
+usage may have a significant impact on performance.
+
+Check the cluster monitoring data to have a better picture of the indexing rate 
+and the memory usage of your cluster. To avoid performance issues, configure 
+forecasting with a small window into the future (for example, a couple of 
+hours), then take action if needed based on the results. If it is necessary, 
+create another forecast with another small window.
+
+Predicting days into the future is not only problematic because of the possible 
+performance issues that it can cause. It is also possible that the analyzed 
+behavior changes significantly over time, making the forecast irrelevant 
+especially for jobs with a short bucket span. As the {anomaly-detect} model is 
+updated constantly, forecasting should be considered as a dynamic process. 
+
+
+[[disabling-model-plot]]
+== Disabling model plot
+
+Model plot calculates and stores the model bounds for each analyzed entity. That 
+means it generates a huge amount of results and has a significant impact on 
+performance. The indexing load is the same as for forecasting: one document per 
+bucket per split field value. For this reason, if you have high cardinality 
+fields and/or short bucket span, disable model plot if it is not needed.
+
+
+[[dedicated-results-index]]
+== Using a dedicated results index
+
+If a {anomaly-job} generates a lot of data, use a dedicated index to store the 
+results. This way, you can retain results in a more reasonable amount of time. 
+If you use a shared index, then the different {anomaly-jobs} slow it down.
```

Comment:
What is `it` in this case? Slow it in what way? There is more than one way for an index to be slow

I think it is better to focus on how the jobs are slowed, not the interactions with the index itself.